vital/bindings/python/vital/tests/helpers.py

```diff
@@ -251,41 +175,39 @@ def reprojection_error_sqr(cam, lm, feat):
     return (reprojection_error_vec(cam, lm, feat) ** 2).sum()
 
 
-def reprojection_rmse(cmap, lmap, tset):
```

Comment:
Why was this removed? It seems unrelated to the goal of this PR. 

If its truly cruft code, I don't mind removing it in this PR, I just want to be sure it is not needed.

Reply:
It requires landmarkMap as the second parameter. I believe landmarkMap bindings were not present in the transition to pybind11 as I can see the c bindings for it but not the python bindings for it. 

I agree that it is not related to this PR but to make helpers.py work LandmarkMap had to be removed from the imports. Since we had multiple helpers that used LandmarkMap as one of the parameters or returned LandmarkMap, I removed them too.